reactnative-todo/src/components/register/text-field.js

```diff
@@ -3,12 +3,14 @@ import { FormControl, Input } from 'native-base';
 import { handleFailedPolicies } from '../utilities/failed-policies';
 
 const TextField = ({ callback }) => {
-  const error = '';
+  const error = handleFailedPolicies(callback.getFailedPolicies());
   const label = callback.getPrompt();
   return (
     <FormControl isInvalid={error}>
       <FormControl.Label>{label}</FormControl.Label>
-      <FormControl.ErrorMessage>{error}</FormControl.ErrorMessage>
+      <FormControl.ErrorMessage>
```

Comment:
Do we want this element to render even when it's just rendering an empty string? Or, should we conditionally render the whole element?